topi/include/topi/scale.h

```diff
@@ -0,0 +1,286 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file topi/scale.h
+ * \brief scaling constructors
+ */
+#ifndef TOPI_SCALE_H_
+#define TOPI_SCALE_H_
+
+#include <string>
+#include <vector>
+#include <iterator>
+#include <algorithm>
+
+#include "topi/tags.h"
+#include "topi/detail/ravel_unravel.h"
+#include "topi/detail/constant_utils.h"
+#include "tvm/tvm.h"
+
+namespace topi {
+using namespace tvm;
+
+/*!
+* \brief Resize given tensor to given shape using nearest neighbour for NHWC
+*
+* \param inputs The input tensor array.
+* \param shape Output shape to scale to.
+* \param align_corners To preserve centers of 4 corner pixels
+* \param name Name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor resized to given shape
+*/
+inline Tensor scale_nn_nhwc(const Array<Tensor>& inputs,
```

Comment:
rename this file to resize, put into subfolder image,

Reply:
Done